enterprise/frontend/src/embedding-sdk/store/selectors.ts

```diff
@@ -27,5 +28,8 @@ export const getErrorComponent = (state: SdkStoreState) =>
 export const getMetabaseInstanceUrl = (state: SdkStoreState) =>
   state.sdk?.metabaseInstanceUrl;
 
+export const getMetabaseInstanceVersion = (state: SdkStoreState) =>
+  state.sdk?.metabaseInstanceVersion ?? getSetting(state, "version")?.tag;
```

Comment:
Nice fallback! If we already have `getSetting(state, "version")?.tag` that stores the version, what's the reason we need a separate `sdk.metabaseInstanceVersion`? Is it because `version.tag` is not always available to us (e.g. the request were made before version.tag is populated?)